test/onnx/exporter/test_small_models_e2e.py

```diff
@@ -509,5 +567,54 @@ def forward(
             self.assertEqual(input.shape[0].value, expected_axis_name)
 
 
+        class ModuleWithACustomOperator(torch.nn.Module):
```

Comment:
Is this a duplicate ?

Reply:
There was merge conflicts. I probably missed something.

Reply:
SG. Looks like everything below can be removed